content/en/docs/concepts/deployment-models/index.md

```diff
@@ -0,0 +1,421 @@
+configuration is known as **multi-network**.
+
+Multiple networks afford the following capabilities beyond that of single networks:
+
+- An existing network topology
+- Overlapping IP or VIP ranges for **service endpoints**
+- Crossing of administrative boundaries
+- Fault tolerance
+- Scaling of network addresses
+
+In this model, the workload instances in different networks can only reach each
+other through one or more [Istio gateways](/docs/concepts/traffic-management/#gateways).
+Istio uses **partitioned service discovery** to provide consumers a different
+view of {{< gloss >}}service endpoint{{< /gloss >}}s. The view depends on the
+network of the consumers.
+
+{{< image width="50%"
+    link="./multi-net.svg"
+    alt="A service mesh with multiple networks"
+    title="Multi-network deployment"
+    caption="A service mesh with multiple networks"
+    >}}
+
+## Mesh models
+
+Istio supports two mesh models:
+
+- Single mesh
+- Multiple meshes, also known as **multi-mesh**
+
+### Single mesh
+
+The simplest Istio deployment is a single mesh. Within a mesh, service names are
+unique. For example, only one service can have the name `foo/mysvc`.
+Additionally, workload instances all share a common identity and authentication
+infrastructure and can authenticate each other. With a single mesh you can do
+the following tasks:
+
+- Span [one or more clusters](#cluster-models)
+- Span [one or more networks](#network-models)
+- Use [namespaces for tenancy](#namespace-tenancy)
+
+### Multiple Meshes
+
+You can deploy multiple meshes; such a configuration is known as
+**multi-mesh**.
+
+Multiple meshes afford the following capabilities beyond that of a single mesh:
+
+- Organizational boundaries: lines of business
+- Service name or namespace collisions: multiple distinct uses of the `default`
+  namespace
+- Stronger isolation: isolating test workloads from production workloads
+
+In the most basic multiple mesh model, each mesh forms a strict
+boundary and there is no communication between meshes. However, you can enable
+inter-mesh communication with **mesh peering**. When
+peering, each mesh can export a set of services and identities, which all
+participating meshes can recognize.
+
+{{< image width="50%"
+    link="./multi-mesh.svg"
+    alt="Multiple service meshes"
+    title="Multi-mesh"
+    caption="Multiple service meshes"
+    >}}
+
+To avoid service naming collisions, you can give each mesh a globally unique
+**mesh ID**, to ensure that the fully qualified domain
+name (FQDN) for each service is distinct.
+
+When peering two meshes that do not share the same
+{{< gloss >}}trust domain{{< /gloss >}}, {{< gloss >}}identity{{< /gloss >}} and
+**trust bundles** must be federated between them. See the section
+on [Multiple Trust Domains](#trust-between-meshes) for an overview.
+
+## Control plane models
+
+The Istio [control plane](https://kubernetes.io/docs/reference/glossary/?fundamental=true#term-control-plane)
```

Comment:
This link to the Kubernetes definition of control plane is not appropriate, since Istio's control plane is not a Kubernetes control plane. We should probably add a generic glossary definition for control plane, and link to it.